Power Up Proto/src/org/usfirst/frc/team5493/robot/commands/TransmissionForward.java

```diff
@@ -12,7 +12,7 @@
 	 private boolean isFinished;
 	 
     public TransmissionForward() {
-    	requires(Robot.throwDaggersInBensEyes);
+//    	requires(Robot.throwDaggersInBensEyes);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)